app/assets/javascripts/analytics/error-tracking.js

```diff
@@ -0,0 +1,24 @@
+// Extension to track errors using google analytics as a data store.
+(function() {
+
+    "use strict";
+    var trackJavaScriptError = function (e) {
+      var errorSource = e.filename + ': ' + e.lineno;
+      _gaq.push([
+        '_trackEvent',
+        'JavaScript Error',
```

Comment:
Just ran this past Ash, he's happy with both the event name (title cased or not) and with experimenting with the idea in principle.